lib/cc/engine/markdownlint.rb

```diff
@@ -33,7 +35,7 @@ def include_paths
         return root unless engine_config.has_key?("include_paths")
 
         markdown_files = engine_config["include_paths"].select do |path|
-          path.end_with?(".md")
+          EXTENSIONS.include?(File.extname(path)) || path.end_with?("/")
```

Comment:
I am a user and I use .markdown, personally. I should open a PR with mdl, though.



~will out.

On Feb 29, 2016, 07:59 -0500, pat brisbin<notifications@github.com>, wrote:
>  
> Inlib/cc/engine/markdownlint.rb(https://github.com/jpignata/codeclimate-markdownlint/pull/1#discussion_r54405898):
>  
> >@@ -33,7 +35,7 @@ def include_paths>return root unless engine_config.has_key?("include_paths")>>markdown_files = engine_config["include_paths"].select do |path|>-     path.end_with?(".md")>+     EXTENSIONS.include?(File.extname(path)) || path.end_with?("/")
>  
> FWIW: I'd be in favor of supporting*.mdonly if that's what the underlying tool does and it simplifies the engine in a meaningful way (which it seems like it would). At least until we see actual user demand for*.markdownsupport.
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/jpignata/codeclimate-markdownlint/pull/1/files#r54405898).
>  
>  
>